### PR TITLE
Emotion2vec

### PR DESCRIPTION
**Pull request by cy3021561**
Update relative functions for emotion2vec model usage.
( emotion2vec feature shape: [TEMPORAL_WINDOW_SIZE, 768] )

1. Add two libraries --> requirement.txt
2. Add emotion2vec model class --> src/emotion2vec/emotion2vec.py
3. Update hidden_layer dimension for emotion2vec --> model/classifier.py
4. Update emotion2vec feature extraction --> preprocess/extract_features.py
5. Add padding and truncate logic to have a fixed emotion2vec feature shape -->dataset/celebv_hq.py

### CODE REVIEW

**Comment by cy3021561** [comment]
Add video feature padding and truncate logic -->dataset/celebv_hq.py

File: small_dataset/emotion2vec/098-0.npy
Comment (adrianSRoman) [inline small_dataset/emotion2vec/098-0.npy:1]: Can we remove this files? They are not part of the repo really.

Comment (cy3021561) [reply small_dataset/emotion2vec/098-0.npy:1]: removed

**Review by adrianSRoman — APPROVED** [review]
Perfect! Thanks master (:

**Pull request merged**